packages/firestore/test/integration/remote/stream.test.ts

```diff
@@ -295,9 +295,11 @@ describe('Write Stream', () => {
           .then(() => {
             // Simulate callback from GRPC with an unauthenticated error -- this should invalidate
             // the token.
-            writeStream.handleStreamClose(
+            return writeStream.handleStreamClose(
               new FirestoreError(Code.UNAUTHENTICATED, '')
             );
+          })
```

Comment:
Is there a reason you added the extra `.then()` instead of using `await`?

Reply:
Yes.  If I changed this then() callback to async/await then I'd feel compelled to convert the whole test, and then I'd feel compelled to convert the whole file, and at that point I may as well go ahead and convert the rest of our codebase.  At that point I'd be so good at it I might as well start tackling other open source projects. And then I'd never get anything done.